### DISCUSSION TITLE
[Question] Would BC412 plastic scintillator be suitable?

### DISCUSSION THREAD

**Discussion by jtomes123** [OpenGammaProject/Open-Gamma-Detector#50] [Q&A]
Hi, would a plastic scintillator be suitable for the detector of would the output be too weak? I can get a cube of 50x50x10mm of BC412 for a reasonable price shipped to EU. I know that plastic scintillator have weaker output than NaI(Tl) or CsI(Tl), however the volume of the scintillator would also be greater. Or if you have other good scintillator source suggestion in EU it would be welcome.

But I like the BC412 since it should be pretty indestructible and moisture resistant.

Thanks!

**Comment by NuclearPhoenixx** [comment]
Hi there, I don't know any scintillator source in the EU. If somebody has one, please let me know as well 😄 

BC412 is okay in theory. However, there are two things to keep in mind:
1. No spectrometry, plastic scintillators have very poor energy resolution so you won't get any useful info out of them.
2. The size you chose is not particularly great for a SiPM. For the best results you'd want a crystal face that has a comparable surface area to the SiPMs, otherwise you'll lose some of the light from the scintillator.

Not to say it wouldn't work just counting pulses, but it's not that ideal. You'll have to try it out. Hope that helps!

**Comment by NuclearPhoenixx** [comment]
Not really an issue, therefore I moved the topic to the Q&A discussions.

**Comment by jtomes123** [comment]
No problem, yeah sadly I don’t have any good place to source the crystals here in EU, they are apparently pretty rare even at my uni which sucks 😅.

The sources I found on ebay were:
LYSO from US (4x4x22mm)
CsI from US (6x6x30mm)
NaI from Ukraine (various but some seemed to be yellowed)

Of the ones I mentioned what do you think would be the best? I am strongly considering the LYSO, since it technically comes with it’s own check source.

**Reply by NuclearPhoenixx** [reply]
Yeah it's a real pain. If I really need to get new ones, I'll get them from China (OST Photonics). Their price is okay, even with added tax and everything and they're brand-new and reliable. I don't know of any real "name-brand", maybe "western" kind of scintillator manufacturer that sells directly to end consumers and less so at affordable prices.

When I started this I got myself two used 18x30mm NaI scintillators from Ukraine, one with 9% and one without any given energy resolution. Both worked and did their job okay. Honestly, I'd have a look at how yellowed the ones from Ukraine are. If they don't look too good, I'd consider the CsI from the US. There really shouldn't be more than just a hint of yellow maybe. Everything else is just bad for performance and suggests there is some kind of air leak which only gets worse with time. I have not personally used LYSO on my system yet, so I wouldn't outright recommend it either.

Also consider that guy right here: https://www.gammaspectacular.com/phpBB3/viewtopic.php?f=14&p=5731#p5731
He used to work a lot with scintillators and refurbished some of them and is apparently based in the netherlands. No idea what he wants for one of them, but he really knows his stuff so send him a mail or something.

**Reply by jtomes123** [reply]
I’ve sent him an email, so we will see, hopefully I can put something together before spring, I am looking to log radiation levels and spectrums on my walks to compile a map of sorts and my childhood rock collection might be also interesting, since the area where I live is known for having more active rocks. The LYSO is intrigueing, but the shipping is more than the crystal and I am also worried about overseas shipping since it is slightly radioactive.

**Reply by NuclearPhoenixx** [reply]
Cool, let me know if you hear from him 😉 

Shipping for LYSO is not a problem, especially these tiny sticks. The activity is waaay below anything that would be considered problematic and it's all "natural" radiation anyways (if that helps). Even with a 1"x1" scintillator up close to two of these LYSO crystals, I only get around 50 cps or so vs 30 background. Nobody's going to bother checking that. But yeah, it got way more expensive. Couple of years ago I could get two of these crystals for like $35 delivered.

**Reply by jtomes123** [reply]
Yeah I missed out on this, I remember I saw SiPM coupled crystals for around 50 USD before the war in ukraine. This LYSO is about 25 USD but the shipping is additional 50.

**Reply by jtomes123** [reply]
I’ve contacted the guy through his email on the forum. He is really helpful and has pretty good prices. I can fully recommend him. I am now waiting for a cut to size CsI crystal 😄. Do you also sell completed sipm boards? I’ve noticed you have carrier boards on tindie.

**Reply by NuclearPhoenixx** [reply]
Very nice. I normally don't sell complete SiPM board, just because of the pretty high inventory cost to stock them. What do you need, though? If you want I can just make you one personally. It's not a lot of trouble to solder the SiPMs if that's what's holding you back, depends on what PCB you want.

**Reply by jbeale1** [reply]
I'm curious what method is best to get more than one SiPM on a board co-planar, without a professional reflow system. My first attempt to do this with hand soldering did not go well, and for the price of these parts it is painful to make mistakes- including that the 99% isopropyl alcohol which I've used without issue on PCBs for several decades, is no good for these delicate parts (although that was my fault for not studying the complete data sheet).

**Comment by dc1rdb** [comment]
I've achieved good results using small reverse tweezers with rubberized tips to fix the SiPMs to the board and then carefully soldering them in place. IMO it is vital to not put any solder on the pads or on the SiPM before placing the SiPM on the board. Coplanarity can then be checked with a suitable light source.
Regarding isopropyl alcohol: been there, done that, bought a new SiPM.

**Reply by NuclearPhoenixx** [reply]
Yeah I'm pretty much doing the same. Put on the SiPMs and then carefully solder one leg at a time making sure that it stays exactly in place using some tweezers. In my experience though, it doesn't matter all _too_ much for the results if they're not 100% exactly co-planar down to like 0.1mm. Just make sure that you're not putting any more pressure onto the SiPMs than you absolutely need to. Optical coupling can take care of your tiny gap. That doesn't mean that they shouldn't be as flat and uniform as possible obviously. @jbeale1 

Also it helps to have ENIG finish on the PCB as that tends to be a lot more flat and uniform on the pads than just your standard HASL.

**Reply by jbeale1** [reply]
Thank you both for these tips! I am ready to construct another SiPM board, so I will try this advice. I would not have guessed it was possible or likely to get a good solder joint without having at least some solder on the pad to start with, but in light of this report I am encouraged to believe it is possible.
